app/views/posts/index.html.erb

```diff
@@ -0,0 +1,52 @@
+<div class="user-navigation">
+  <a href="<%= user_path(@user) %>">
+    <button class="back-btn">⬅</button>
+  </a>
+</div>
+<section class="user-profile-page">
+  <div class='post-user-info'>
+    <% if @user.photo.present? %>
+      <%= image_tag @user.photo, alt: @user.name, class: 'user-thumbnail' %>
+    <% else %>
+      <span class="user-thumbnail placeholder">No Photo</span>
+    <% end %>
+    <p>Name: <%= @user.name %></p>
+    <p>Number of Posts: <%= @user.posts.count %></p>
+  </div>
+  <div class="user-posts">
+    <% if @user.posts.any? %>
+      <ul class="user-posts-info">
+        <% @user.posts.limit(3).each do |post| %>
+          <li>
+            <%= link_to user_post_path(@user, post) do %>
+              <h3><%= post.title %></h3>
+              <p><%= post.text %></p>
+            <% end %>
+          </li>
+
+          <div class="counters">
+            <li>Comments: <%= post.comments_counter %></li>
+            <li>Likes: <%= post.likes_counter %></li>
+          </div>
+
+          <% post.comments.each do |comment| %>
```

Comment:
- [ ] I think here  you're supposed to call the `#most_recent_comments` method on the post to render the 5 most recent comments as requested :+1: